app/helpers/hyrax_helper.rb

```diff
@@ -19,4 +19,8 @@ module HyraxHelper
   include ::BlacklightHelper
   include Hyrax::BlacklightOverride
   include Hyrax::HyraxHelperBehavior
+
+  def capitalize_value(value)
+    value.capitalize
```

Comment:
I think the overall lower coverage could be fixed if you added a simple unit test for this method.